include/NovelRT.h

```diff
@@ -57,8 +57,8 @@
 #include <glm/gtx/rotate_vector.hpp>
 
 //OpenAL
-#include <AL/al.h>
-#include <AL/alc.h>
+#include "al.h"
+#include "alc.h"
```

Comment:
Won't this break out-of-conan builds?

Reply:
Possibly - I'm unsure how these got changed in the first place. I'll see about changing them back.